cluster-api/src/main/java/com/sequenceiq/cloudbreak/repository/ClusterComponentHistoryRepository.java

```diff
@@ -0,0 +1,19 @@
+package com.sequenceiq.cloudbreak.repository;
+
+import javax.transaction.Transactional;
+import javax.transaction.Transactional.TxType;
+
+import java.util.Collection;
+
+import com.sequenceiq.cloudbreak.domain.stack.cluster.ClusterComponentHistory;
+import com.sequenceiq.cloudbreak.workspace.repository.EntityType;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.lang.NonNull;
+
+@Transactional(TxType.REQUIRED)
+@EntityType(entityClass = ClusterComponentHistory.class)
+public interface ClusterComponentHistoryRepository extends CrudRepository<ClusterComponentHistory, Long> {
```

Comment:
I don't really get this class, neither the ClusterComponentHistory.
It is a brand new entity, what is going to fill it? If it did not exist before, why is it needed to clean it up?